config/locales/pages/data_sharing_agreement/en.yml

```diff
@@ -145,7 +145,7 @@ en:
           trainees how their data will be used. This privacy notice will
           include the email address they can use to contact the Register
           support team, and can be found at 
-          <a class='govuk-link' href='https://www.register-trainee-teachers.education.gov.uk/privacy-policy'>www.register-trainee-teachers.education.gov.uk/privacy-policy</a>
+          <a class='govuk-link' href='https://www.register-trainee-teachers.education.gov.uk/privacy-notice'>www.register-trainee-teachers.education.gov.uk/privacy-notice</a>
```

Comment:
looks good to me, I couldn't really check this specific link as it's a prod link and the notice route isn't generated yet - but visiting it directly with the review app URL works fine

Reply:
will check once it's on prod 👍